### PR TITLE
The Runmons Reckoning

### PR DESCRIPTION
**Pull request by red031000**
Reopening PR #39

### CODE REVIEW

**Comment by red031000** [comment]
it appears that a conflict was introduced here due to the merge fuckery, if that could be resolved that'd be great

File: pokesets/Runmon/12_PuppetPlay.yaml [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+gender: [f]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Poke]
+shiny: False
+nature: Quiet
+ivs: {hp: 21, atk: 2, def: 17, spA: 21, spD: 10, spe: 5}
+evs: {hp: 155, atk: 57, def: 22, spA: 111, spD: 44, spe: 121}
+
```
Comment (MathyFurret) [inline pokesets/Runmon/12_PuppetPlay.yaml:21]: Missing --- between sets here too.

File: pokesets/Runmon/17_Brown.yaml [hunk 2]
```diff
@@ -101,6 +101,27 @@ happiness: 255
 biddable: True
 rarity: 1.0
 shiny: False
-nature: Timid
-ivs: 0
-evs: 40
+nature: Serious
+ivs: {hp: 2, atk: 1, def: 4, spA: 10, spD: 10, spe: 4}
+evs: {hp: 127, atk: 138, def: 130, spA: 125, spD: 125, spe: 147}
+
```
Comment (MathyFurret) [inline pokesets/Runmon/17_Brown.yaml:107]: Still need a --- here

File: pokesets/Runmon/47_Sirius.yaml [hunk 3]
```diff
@@ -62,3 +62,27 @@ shiny: False
 nature: Quiet
 ivs: {hp: 7, atk: 25, def: 18, spA: 18, spD: 1, spe: 5}
 evs: {hp: 2, atk: 4, def: 0, spA: 0, spD: 0, spe: 0}
+
```
Comment (MathyFurret) [inline pokesets/Runmon/47_Sirius.yaml:65]: --- here

**Review by MathyFurret — CHANGES_REQUESTED** [review]
I think you fixed the error in some places but it's still in others (the compile script is silent about it and i sort of know why, but i dont know the semantics about duplicated keys in yaml)

**Comment by MathyFurret** [comment]
Pinging @SolarisLumos ^^^ because i forgot he wont get notifs anymore

**Comment by MathyFurret** [comment]
thank you so much! and sorry about all the delays

**Pull request merged**